### PR TITLE
Add some initial commands

### PR DESCRIPTION
**Pull request by alessandrojean**
Add lock and duplicate commands.

They still need to be tested.

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  }
+}
+
+async function lockIssue(client: GitHubClient) {
+  const { payload, repo } = github.context;
+  const commentBody: string = payload.comment.body;
+
+  const lockReasons = ['off-topic', 'too heated', 'resolved', 'spam'];
+
+  const reason = lockReasons.find(option => commentBody.includes(option));
```
Comment (arkon) [inline src/main.ts:67]: Doesn't seem particularly intuitive. Maybe at least document it.

File: src/main.ts [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+  await client.rest.issues.lock({
+    owner: repo.owner,
+    repo: repo.repo,
+    issue_number: payload.issue.number,
+    lock_reason: reason ? reason as LockReason : undefined
```
Comment (arkon) [inline src/main.ts:73]: Is this just to deal with type issues? Seems like an unnecessary ternary.

**Review by arkon — APPROVED** [review]
Seems fine

File: README.md [hunk 3]
```diff
@@ -1 +1,31 @@
...
+name: Moderator
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  moderate:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Moderate issues
+      uses: tachiyomiorg/issue-moderator-action
```
Comment (arkon) [inline README.md:20]: Missing version. Unless that's actually optional?

**Comment by alessandrojean** [comment]
I merged at the same time when you're reviewing, sorry. I will change the things in a next PR.

File: src/main.ts (hunk 2 above)
Comment (alessandrojean) [reply src/main.ts:73]: Yep, to deal with some type issues.

File: README.md (hunk 3 above)
Comment (alessandrojean) [reply README.md:20]: Don't think so, I forgot to add it.

**Pull request merged**